sgkit_plink/pysnptools.py

```diff
@@ -122,13 +121,13 @@ def coerce_code(v, codes):
 def read_bim(path: PathType, sep: str = "\t") -> DataFrame:
     # See: https://www.cog-genomics.org/plink/1.9/formats#bim
     names = [f[0] for f in BIM_FIELDS]
-    df = dd.read_csv(str(path) + ".bim", sep=sep, names=names, dtype=BIM_DF_DTYPE)
+    df = dd.read_csv(str(path), sep=sep, names=names, dtype=BIM_DF_DTYPE)
     df["contig"] = df["contig"].where(df["contig"] != "0", None)
     return df
 
 
 def read_plink(
-    path: PathType,
+    path: Union[PathType, Tuple[PathType, PathType, PathType]],
```

Comment:
This change is unrelated to slashes, also has unaddressed issue from #11

Reply:
I'll merge the changes from https://github.com/pystatgen/sgkit-plink/pull/15 whenever it goes through.